analyses/molecular-subtyping-pathology/results/compiled_molecular_subtypes.tsv

```diff
@@ -3,101 +3,113 @@ PT_00G007DM	7316-272	BS_QWNBZ9RJ	ETMR, C19MC-altered	Embryonal tumor with multil
 PT_00G007DM	7316-272	BS_K07KNTFY	ETMR, C19MC-altered	Embryonal tumor with multilayer rosettes, C19MC-altered	ETMR	Embryonal tumor	Updated via OpenPBTA subtyping
 PT_01MZ62KG	7316-447	BS_P39SQPTS	ETMR, C19MC-altered	Embryonal tumor with multilayer rosettes, C19MC-altered	ETMR	Embryonal tumor	Updated via OpenPBTA subtyping
 PT_01MZ62KG	7316-447	BS_TE8QFF7T	ETMR, C19MC-altered	Embryonal tumor with multilayer rosettes, C19MC-altered	ETMR	Embryonal tumor	Updated via OpenPBTA subtyping
-PT_02J5CWN5	7316-2989	BS_M8WP5T16	LGG, FGFR	Low-grade glioma/astrocytoma, FGFR	LGAT	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
-PT_02J5CWN5	7316-2989	BS_2VB8649B	LGG, FGFR	Low-grade glioma/astrocytoma, FGFR	LGAT	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
-PT_02J5CWN5	7316-898	BS_1RXZ3BP7	LGG, wildtype	Low-grade glioma/astrocytoma, wildtype	LGAT	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
-PT_02J5CWN5	7316-898	BS_5TT6TT4K	LGG, wildtype	Low-grade glioma/astrocytoma, wildtype	LGAT	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
+PT_02J5CWN5	7316-2989	BS_M8WP5T16	LGG, FGFR	Low-grade glioma/astrocytoma, FGFR	Low-grade astrocytic tumor	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
+PT_02J5CWN5	7316-2989	BS_2VB8649B	LGG, FGFR	Low-grade glioma/astrocytoma, FGFR	Low-grade astrocytic tumor	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
+PT_02J5CWN5	7316-898	BS_1RXZ3BP7	LGG, wildtype	Low-grade glioma/astrocytoma, wildtype	Low-grade astrocytic tumor	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
+PT_02J5CWN5	7316-898	BS_5TT6TT4K	LGG, wildtype	Low-grade glioma/astrocytoma, wildtype	Low-grade astrocytic tumor	Low-grade astrocytic tumor	Updated via OpenPBTA subtyping
```

Comment:
@runjin326 can you check into why this is bringing in a longer histology for the short histology?